src/runtime/graph.cc

```diff
@@ -1129,20 +1149,46 @@ GraphCostResult GraphCostResult::invalid() {
   return {std::numeric_limits<float>::infinity(), {}};
 }
 
+GraphCostResultWithMemory GraphCostResultWithMemory::invalid() {
+  return {std::numeric_limits<float>::infinity(), MemoryUsage{}, {}};
```

Comment:
Is there any way we could make this more readable? That's a lot of braces with unclear meanings.

Reply:
Hmm, I was just making it consistent with the previous implementation above.

Reply:
Yeah, just as there are more fields it gets harder to track/remember everything. Not a big deal though